src/renderer/screens/exchange/Swap2/Form/FormSummary/SectionFees.js

```diff
@@ -25,6 +55,10 @@ const SectionFees = ({ swapTransaction }: { swapTransaction: SwapTransactionType
   const showSummaryValue = fromAccountUnit && estimatedFees && estimatedFees.gt(0);
   const canEdit =
     showSummaryValue && transaction?.networkInfo && sendAmountByFamily[account?.currency?.family];
+  const StrategyIcon = useMemo(
+    () => FEES_STRATEGY_ICONS[swapTransaction.transaction?.feesStrategy],
+    swapTransaction.transaction?.feesStrategy,
```

Comment:
If I read correctly 👓 , did you forgot the brackets? `[swapTransaction.transaction?.feesStrategy]`

Reply:
yeah, I did, nice catch